### DISCUSSION TITLE
wave doesn't show up

### DISCUSSION THREAD

**Discussion by yeonatanPerelman** [FPGA-MAFIA/fpga_mafia#29] [Q&A] [answered]
Hi Amichai,
we try to run the modelsim, but we see the next comment:
![צילום מסך 2022-11-20 111040](https://user-images.githubusercontent.com/111231093/202894233-0a6f807d-2fc0-4424-a3a3-f7646abf693f.png)
Warning: (vsim-WLF-5000) WLF file currently in use: vsim.wlf
#           File in use by: 97254  Hostname: LAPTOP-0TNQ97T0  ProcessID: 4256
#           Attempting to use alternate WLF file "./wlft2ykne4".
# ** Warning: (vsim-WLF-5001) Could not open WLF file: vsim.wlf
#           Using alternate file: ./wlft2ykne4

we try to rerun it again on the vscode, but nothing change.

**Comment by amichai-bd** [comment]
Let's review this together.
Please text me on whatsapp and we will try and get it to work together

**Comment by amichai-bd** [comment]
Please Document what the issue was and how you resolved it in this thread.
@yeonatanPerelman 
Then mark this as "answered"

**Comment by amichai-bd** [comment, marked as answer]
@noamsabb explain what went wrong and how you solved it.

**Reply by noamsabb** [reply]
When i ran a simulation the code was runing good on the terminal (in VSCode) but wasn't showing waves in VSIM
To fix it i had to write the **restart** command in the VSIM Transcript window
![Capture d’écran 2022-11-28 à 15 12 47](https://user-images.githubusercontent.com/83982485/204286328-1b18a79e-c294-497f-b882-e4a65253c86a.png)
then i had to set the time to a good value because it was reset to 0 ps
![Capture d’écran 2022-11-28 à 15 13 42](https://user-images.githubusercontent.com/83982485/204286509-b500d632-01e4-4d7c-8b3c-e3651b9337c0.png)
then write the **run** command in the VSIM Transcript window or press the run button (next to the time set) and waves started to show
